src/Autohide.js

```diff
@@ -14,6 +14,7 @@ define(function (require, exports, module) {
     var $mainView = $(".main-view");
     var isExposed = false;
     var sidebarPlusTransitionRemoved = false;
+    var scrollBackup;
```

Comment:
Could you maybe rename this to `scrollTop`, `previousScrollTop`, or something similar?

Reply:
First I wrote `scrollTop` and then dismissed it as redundant, hehe

Renamed.